README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
+## Build Instructions 
+
+BUILD INSTRUCTIONS
```

Comment:
Why does the section labeled "Build Instructions" have an opening section labeled "BUILD INSTRUCTIONS"?  It seems odd and it seems like this line is superfluous.